[SMS] The parameter conversion failed for SMS task resource

Hi team,

I found a problem in the resource `huaweicloud_sms_task` , the panic message is as follows:
```
Plan: 1 to add, 0 to change, 0 to destroy.
huaweicloud_sms_task.sms_task: Creating...
╷
│ Error: Plugin did not respond
│ 
│   with huaweicloud_sms_task.sms_task,
│   on main.tf line 4, in resource "huaweicloud_sms_task" "sms_task":
│    4: resource "huaweicloud_sms_task" "sms_task" {
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange
│ call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-huaweicloud_v1.54.1 plugin:

panic: interface conversion: interface {} is int, not int64

goroutine 145 [running]:
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/sms.buildTargetServerDiskRequest(0x1727660?, 0x40011bafc0?, {0x4000a44000, 0x24})
        github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/sms/resource_huaweicloud_sms_task.go:313 +0x3a4
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/sms.buildTargetServerRequest(0x1de8628?, 0x4000576b40, {0x4000a44000, 0x24})
        github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/sms/resource_huaweicloud_sms_task.go:342 +0x380
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/sms.buildMigrateTaskRequest(0x4000576b40?, 0x4000576b40)
        github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/sms/resource_huaweicloud_sms_task.go:386 +0xac
github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/sms.resourceMigrateTaskCreate({0x1dfb488, 0x4000a5d020}, 0x0?, {0x1a7e500?, 0x4000576b40?})
        github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/sms/resource_huaweicloud_sms_task.go:428 +0x144
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0x4000ced0a0, {0x1dfb4c0, 0x4000a13980}, 0xd?, {0x1a7e500, 0x4000576b40})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.0/helper/schema/resource.go:707 +0xe8
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0x4000ced0a0, {0x1dfb4c0, 0x4000a13980}, 0x4000573040, 0x400056d200, {0x1a7e500, 0x4000576b40})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.0/helper/schema/resource.go:837 +0x85c
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0x40004aa0c0, {0x1dfb4c0?, 0x4000a13860?}, 0x400128ea00)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.0/helper/schema/grpc_provider.go:1021 +0xb5c
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0x4000f8a320, {0x1dfb4c0?, 0x4000a13050?}, 0x40002042a0)
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/tf5server/server.go:818 +0x3b8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x19df200?, 0x4000f8a320}, {0x1dfb4c0, 0x4000a13050}, 0x4000204230, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.14.
```

pls help to fix it.

there is my execution plan:
```
# huaweicloud_sms_task.sms_task will be created
  + resource "huaweicloud_sms_task" "sms_task" {
      + enterprise_project_id = (known after apply)
      + id                    = (known after apply)
      + migrate_speed         = (known after apply)
      + migration_ip          = "123.249.35.45"
      + os_type               = "LINUX"
      + project_id            = (known after apply)
      + region                = (known after apply)
      + source_server_id      = "2178dfb5-1adf-43cc-94a8-829e35e93e2e"
      + start_target_server   = true
      + state                 = (known after apply)
      + target_server_id      = "8304d0d9-d33e-4952-8328-1b2ed373c86a"
      + target_server_name    = (known after apply)
      + type                  = "MIGRATE_FILE"
      + use_public_ip         = true

      + target_server_disks {
          + device_type = "NORMAL"
          + disk_id     = (known after apply)
          + name        = "zhang"
          + size        = 40
          + used_size   = (known after apply)

          + physical_volumes {
              + device_type = "NORMAL"
              + file_system = "ext4"
              + index       = 1
              + mount_point = "/"
              + name        = "zhangpv"
              + size        = 100
              + used_size   = (known after apply)
              + uuid        = (known after apply)
            }
        }
    }
```

--- a/docs/resources/sms_task.md
+++ b/docs/resources/sms_task.md
@@ -83,7 +83,7 @@ The `target_server_disks` block supports:
 * `device_type` - (Required, String, ForceNew) Specifies the disk type. The value can be **NORMAL** and **BOOT**.
   Changing this parameter will create a new resource.
 
-* `disk_id` - (Optional, String, ForceNew) Specifies the disk index, e.g. "0".
+* `disk_id` - (Required, String, ForceNew) Specifies the disk index, e.g. "0".
   Changing this parameter will create a new resource.
 
 * `used_size` - (Optional, Int, ForceNew) Specifies the used space in MB. Changing this parameter will create a new resource.

--- a/huaweicloud/services/sms/resource_huaweicloud_sms_task.go
+++ b/huaweicloud/services/sms/resource_huaweicloud_sms_task.go
@@ -111,10 +111,11 @@ func ResourceMigrateTask() *schema.Resource {
 							ForceNew: true,
 						},
 						"disk_id": {
-							Type:     schema.TypeString,
-							Optional: true,
-							Computed: true,
-							ForceNew: true,
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							ForceNew:    true,
+							Description: "schema: Required",
 						},
 						"used_size": {
 							Type:     schema.TypeInt,

--- a/huaweicloud/services/sms/resource_huaweicloud_sms_task.go
+++ b/huaweicloud/services/sms/resource_huaweicloud_sms_task.go
@@ -284,13 +284,13 @@ func buildTargetServerPVRequest(raw []interface{}) []tasks.PVRequest {
 		idx := item["index"].(int)
 		pvs[i] = tasks.PVRequest{
 			Name:       item["name"].(string),
-			Size:       convertMBtoBytes(item["size"].(int64)),
+			Size:       convertMBtoBytes(int64(item["size"].(int))),
+			UsedSize:   convertMBtoBytes(int64(item["used_size"].(int))),
 			DeviceType: item["device_type"].(string),
 			FileSystem: item["file_system"].(string),
 			MountPoint: item["mount_point"].(string),
 			Index:      &idx,
 			UUID:       item["uuid"].(string),
-			UsedSize:   convertMBtoBytes(item["used_size"].(int64)),
 		}
 	}
 
@@ -310,8 +310,8 @@ func buildTargetServerDiskRequest(d *schema.ResourceData, cfg *config.Config, si
 		disks[i] = tasks.DiskRequest{
 			Name:            item["name"].(string),
 			DeviceType:      item["device_type"].(string),
-			Size:            convertMBtoBytes(item["size"].(int64)),
-			UsedSize:        convertMBtoBytes(item["used_size"].(int64)),
+			Size:            convertMBtoBytes(int64(item["size"].(int))),
+			UsedSize:        convertMBtoBytes(int64(item["used_size"].(int))),
 			DiskId:          item["disk_id"].(string),
 			PhysicalVolumes: buildTargetServerPVRequest(item["physical_volumes"].([]interface{})),
 		}

